src/platforms/web/server.js

```diff
@@ -87,7 +89,7 @@ app.get('*', async (req, res) => {
             ? truncateDescription(claim.description)
             : `Watch ${title} on LBRY.tv`;
         const claimLanguage = claim.language || 'en_US';
-        const claimThumbnail = claim.thumbnail_url || '/og.png';
+        const claimThumbnail = claim.thumbnail_url || 'https://beta.lbry.tv/og.png';
```

Comment:
same comments as before.  I think this PR is invalid. Not sure what the fb:app_id does - @jsigwart ?

Reply:
facebook tracking stuff to help josh find new users. :)